src/sentry/api/endpoints/api_tokens.py

```diff
@@ -101,7 +103,7 @@ def post(self, request: Request) -> Response:
                 send_email=True,
             )
 
-            analytics.record("api_token.created", user_id=request.user.id)
+            analytics.record(ApiTokenCreated(user_id=request.user.id))
```

Comment:
`request.user.id` is typed as `Any | None`, so idk whether passing it here is safe